src/html/erc20-form.html

```diff
@@ -215,15 +215,24 @@ <h2 class="modal-section__label">Select from whitelist</h2>
     });
   });
 
-  async function fillFeaturedErc20s() {
+  async function fillErc20List () {
     if (!window.ethInitialized) return;
     if (!window.isValidEthNetwork) return;
     if (window.urlParams.get("erc20") === null) return;
 
-    const featured = await window.utils.getFeaturedErc20s();
+    const allTokens = await window.utils.getAllErc20s();
+
+    function ownedTokenOnTop (token1, token2) {
```

Comment:
No reason to re-define this function on every run of `fillErc20List`; it can be moved outside